californium-core/src/main/java/org/eclipse/californium/core/network/InMemoryMessageExchangeStore.java

```diff
@@ -81,8 +97,8 @@ public InMemoryMessageExchangeStore(final NetworkConfig config, TokenProvider to
 		}
 		this.tokenProvider = tokenProvider;
 		this.config = config;
-		this.tokenLength = config.getInt(NetworkConfig.Keys.TOKEN_SIZE_LIMIT, MAX_TOKEN_LENGTH);
-		LOGGER.log(Level.CONFIG, "using tokens of {0} bytes in length", tokenLength);
+		LOGGER.log(Level.CONFIG, "using tokens of {0} bytes in length",
```

Comment:
No need to log this here. Since the `TokenProvider` is using this config param, it should be logged out in the `InMemoryRandomTokenProvider` instead ...